__tests__/database/migration/index.test.ts

```diff
@@ -5,7 +5,7 @@ describe('TESTING migration file', () => {
     expect(DROP_ALL_DB).toBeFalsy();
   });
 
-  it('TEST should expect TARGET_MIGRATION_VERSION to be 8', () => {
-    expect(TARGET_MIGRATION_VERSION).toBe(8);
+  it('TEST should expect TARGET_MIGRATION_VERSION to be 15', () => {
+    expect(TARGET_MIGRATION_VERSION).toBe(15);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test case is checking if `TARGET_MIGRATION_VERSION` is 15. If the migration version has been updated to 15 in the codebase, this change is correct. However, if the migration version hasn't been updated or is different, this test will fail. Please ensure that the `TARGET_MIGRATION_VERSION` in your database configuration matches the expected value in this test.
```diff
-    expect(TARGET_MIGRATION_VERSION).toBe(15);
+    expect(TARGET_MIGRATION_VERSION).toBe(<your_updated_migration_version>);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->